domain-ee/ee-ep-merge-app/src/python_src/util/metric_logger.py

```diff
@@ -0,0 +1,56 @@
+import logging
+
+import datadog
+from config import ENV
+
+APP_PREFIX = 'ep_merge'
+
+ENV_TAG = f'environment:{ENV}'
+SERVICE_TAG = 'service:vro-ee-ep-merge-app'
+STANDARD_TAGS = [ENV_TAG, SERVICE_TAG]
+
+
+def start():
+    logging.info('Datadog DogStatsD initializing...')
+    datadog.initialize()
+    logging.info('Datadog DogStatsD initialized.')
+
+
+def stop():
+    logging.info('Datadog DogStatsD flushing...')
+    datadog.statsd.flush()
+    logging.info('Datadog DogStatsD flushed. Closing socket...')
+    datadog.statsd.close_socket()
+    logging.info('Datadog DogStatsD socket closed.')
+
+
+def increment(metric: str, value: float = 1, tags: list = None):
+    """
+    Increments a count metric with by the name 'APP_PREFIX.{metric}'
+    :param metric: string containing the metric name
+    :param value: value to increment by
+    :param tags: any associated tags with the metric
+    """
+
+    if not tags:
+        tags = STANDARD_TAGS
+    else:
+        tags.extend(STANDARD_TAGS)
```

Comment:
It doesn't look like any calls to `increment` in this PR use extra tags, but I feel like you could default it to `= []` and then just pass `tags + STANDARD_TAGS` to statsd.

Reply:
Ah yeah. That's what I had to start but intelliJ was giving me a warning about using a mutable default argument. 🤷‍♂️

Reply:
Yeah, it's a valid warning, but for short functions it's just as easy to eyeball the code and see that the default isn't getting mutated. Or one clunky way to squash the warning is to default to immutable `()` and then evaluate `list(tags) + STANDARD_TAGS`.

But I'm perfectly fine with removing it altogether. 😄